### DISCUSSION TITLE
Add option to remove authors that are members of the same github org

### DISCUSSION THREAD

**Discussion by srenatus** [community/community#6032] [Other Feature Feedback, Questions, & Ideas] [answered]
We tend not to call out authors that are already part of the team, but only include _"by @nosuchname"_ if the user isn't.

It would be nice if this was an option somewhere.

**Comment by MylesBorins** [comment, marked as answer]
hey @srenatus. We strongly believe that it is important to call out all authors independent if they are from the org or not. This is important from a recognition stand point as well as from a work auditing perspective. At this time this is not a feature that we would be supporting.
